src/creatures/monsters/monster.cpp

```diff
@@ -2006,7 +2006,6 @@ void Monster::dropLoot(Container* corpse, Creature*) {
 				corpse->internalAddThing(sliver);
 			}
 		}
-		g_events().eventMonsterOnDropLoot(this, corpse);
```

Comment:
My only concern is not removing this (yet) in order to maintain compatibility with older servers, should they choose not to adopt the revscript event.